wallets/react-web3wallet/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "react-web3wallet",
+  "private": true,
+  "scripts": {
+    "dev": "next dev -p 3001",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "prettier:write": "prettier --write '**/*.{js,ts,jsx,tsx}'"
+  },
+  "dependencies": {
+    "@cosmjs/amino": "0.28.4",
+    "@cosmjs/encoding": "0.28.4",
+    "@cosmjs/proto-signing": "0.28.4",
+    "@elrondnetwork/erdjs": "11.1.1",
+    "@elrondnetwork/erdjs-walletcore": "2.1.0",
+    "@json-rpc-tools/utils": "1.7.6",
+    "@near-wallet-selector/wallet-utils": "^7.0.2",
+    "@nextui-org/react": "1.0.8-beta.5",
+    "@polkadot/keyring": "^10.1.2",
+    "@polkadot/types": "^9.3.3",
+    "@solana/web3.js": "1.43.0",
+    "@walletconnect/core": "2.1.4",
+    "@walletconnect/sign-client": "2.1.4",
+    "@walletconnect/utils": "2.1.4",
+    "@walletconnect/web3wallet": "^1.0.0-rc-ed782cef",
```

Comment:
Question: Do sign and core dependencies still need to be installed or are they pre-bundled into web3wallet package?